transports/janus_mqtt.c

```diff
@@ -1612,7 +1612,9 @@ static gboolean janus_mqtt_vacuum(gpointer context) {
 
 	while (g_hash_table_iter_next(&iter, NULL, &value)) {
 		janus_mqtt_transaction_state* state = value;
-		if(janus_get_monotonic_time() - state->created_at > ctx->vacuum_interval) {
+		gint64 diff = (janus_get_monotonic_time() - state->created_at) / G_GINT64_CONSTANT(1000000);
+
```

Comment:
It may be better to use `G_USEC_PER_SEC` instead of `1000000` here, just for readability. As a side note, please remove the extra space above, I don't think it's needed.